docs/UserGuide.md

```diff
@@ -154,6 +158,28 @@ Examples:
 * `editi 3 i/2 t/screening` edits the 2nd interview type of the 3rd person in the displayed applicant list to a screening interview.
 * `editi 2 i/1 r/8.9` edits the 1st interview rating of the 2nd person in the displayed applicant list to 8.9.
 
+---
+### `deletei` : Deleting an interview from an applicant
+
+Deletes an interview from an applicant.
+
+Format: `deletei INDEX i/INTERVIEW_INDEX`
+
+Examples:
+* `deletei 1 i/2` deletes the 2nd interview of the 1st person in the displayed applicant list.
+
+---
+### `status` : Editing an applicant status
+
+Clears all the current data stored in the system.
```

Comment:
Just a minor mistake, `status` edits an applicant status instead of clearing data 😸